modules/virt-importing-vm-datavolume.adoc

```diff
@@ -106,44 +106,46 @@ spec:
-<2> The source of the virtual machine image you want to import. This example references a virtual machine image at an HTTP endpoint. An example of a container registry endpoint is `url: "docker://kubevirt/fedora-cloud-container-disk-demo:latest"`.
-<3> The `secretRef` parameter is optional.
-<4> The `certConfigMap` is required for communicating with servers that use self-signed certificates or certificates not signed by the system CA bundle. The referenced config map must be in the same namespace as the data volume.
-<5> Specify `type: dataVolume` or `type: ""`. If you specify any other value for `type`, such as `persistentVolumeClaim`, a warning is displayed, and the virtual machine does not start.
+<1> Specify the name of the virtual machine
+<2> Specify the name of the data volume.
+<3> Specify `http`, for an HTTP or HTTPS endpoint, or `registry`, for a container disk image imported from a registry.
+<4> The source of the virtual machine image you want to import. This example references a virtual machine image at an HTTPS endpoint. An example of a container registry endpoint is `url: "docker://kubevirt/fedora-cloud-container-disk-demo:latest"`.
+<5> Required if you created a `Secret` for the data source.
+<6> Optional: Specify a CA certificate config map.
+====
 
 . Create the virtual machine:
 +
 [source,terminal]
 ----
-$ oc create -f vm-<name>-datavolume.yaml
+$ oc create -f <vm_fedora_datavolume>.yaml
 ----
 +
 [NOTE]
 ====
-The `oc create` command creates the data volume and the virtual machine. The CDI controller creates an underlying PVC with the correct annotation, and the import process begins. When the import completes, the data volume status changes to `Succeeded`, and the virtual machine is allowed to start.
+The `oc create` command creates the data volume and the virtual machine. The CDI controller creates an underlying PVC with the correct annotation and the import process begins. When the import is complete, the data volume status changes to `Succeeded`. You can start the virtual machine.
 
-Data volume provisioning happens in the background, so there is no need to monitor it. You can start the virtual machine, and it will not run until the import is complete.
+Data volume provisioning happens in the background, so there is no need to monitor the process.
 ====
 
 .Verification
+
 . The importer pod downloads the virtual machine image or container disk from the specified URL and stores it on the provisioned PV. View the status of the importer pod by running the following command:
 +
 [source,terminal]
 ----
 $ oc get pods
 ----
 
-. Monitor the data volume status until it shows `Succeeded` by running the following command:
+. Monitor the data volume until its status is `Succeeded` by running the following command:
 +
 [source,terminal]
 ----
-$ oc describe dv <datavolume-name> <1>
+$ oc describe dv <fedora-dv> <1>
```

Comment:
If you're using this as an example (which the context suggests), remove the < > around "fedora-dv"